src/axom/slic/interface/slic.hpp

```diff
@@ -313,17 +312,52 @@ void logWarningMessage(const std::string& message,
                        int line);
 
 /*!
- * \brief Flushes all streams.
+ * \brief For the current rank, outputs messages from all streams to the
+ *        console
+ *
+ * \warning outputLocalMessages() is used before a rank aborts.
+ *          flushStreams() is preferred over this function,
+ *          as outputLocalMessages() may put LogStreams in an undesirable
+ *          state. This call is not collective.
+ */
+void outputLocalMessages();
+
+///@{
+//! \name Collective Methods
+//!
+//! \attention These methods are collective operations.
+//! All ranks in the user-supplied communicator must call the method
+//! when used within an MPI distributed environment.
+//!
+//! \sa axom::slic::isAbortOnErrorsEnabled()
```

Comment:
Is calling these collective correct? I know they have to be called on each node to be correct but there is no communication, its just getting/setting a local boolean.

Reply:
Good catch.